### ISSUE TITLE
Concatenate Distinct Duplicating Values 

### ISSUE THREAD

**Issue by escholl** [afawcett/declarative-lookup-rollup-summaries#840]
I've used this function before without issue. I'm using Operation = Concatenate Distinct to Concatenate values separated by ";" on the Opportunity object to roll up to a field on the Account without duplication. Values are being duplicated and I can't find a work around. See screen shot: 
![Screen Shot 2019-07-02 at 12 31 15 PM](https://user-images.githubusercontent.com/44210610/60537264-5fb99900-9cc5-11e9-8e0a-dc3738099a31.png)
![Screen Shot 2019-07-02 at 12 31 32 PM](https://user-images.githubusercontent.com/44210610/60537265-5fb99900-9cc5-11e9-890e-dcf3a7c8afa0.png)

**Comment by JimBTek** [comment]
Is the field you are aggregating multiselect?
Have you tried Sharing Mode as System?
Have you tried adding the field to aggregate to the Relationship Criteria fields?
Are they definitely exact match values on the two records if you inspect them?

**Comment by escholl** [comment]
Hey Jim, thanks for the reply! 
Yes it's a MSP and I've tried all of those things. I think it's because I'm trying to keep historical data. 
Example: We have an MSP on the Account object for Customer Use Case (why the customer came to us) and an MSP on the Opportunity with the same values. When an Opportunity is close/won, I want the Use Case MSP values from the won Opportunity to roll up to the Account 1 - Without duplication and 2 - Without erasing any manual data on the Account MSP. 
I've tried using a formula field to combine the historical data and the Opportunity MSP and then DLRS to roll up to the Account MSP and it works, but, there's duplication. Any advice?
